packages/playground/src/components/samples-dropdown.tsx

```diff
@@ -9,6 +9,17 @@ export const SamplesDropdown: FunctionComponent<SamplesDropdownProps> = ({ onSel
     return <option key={sample}>{sample}</option>;
   });
 
+  useEffect(() => {
+    if (window.location.search.length > 0) {
+      const parsed = new URLSearchParams(window.location.search);
+      const sample = parsed.get("sample");
+      if (sample) {
+        setSelected(sample);
+        onSelectSample(PlaygroundManifest.samples[sample]);
+      }
+    }
```

Comment:
I would be good to also maybe add a ui test for that in `ui.e2e.ts` should be quite similar to the "shared link works" test